src/utils/resmoke/index.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  getAttributes,
+  getConfigServer,
+  getContext,
+  getId,
+  getJSONString,
+  getMessage,
+  getPort,
+  getResmokeFunction,
+  getShellPrefix,
+  getTimeStamp,
+} from "./utils";
+
+const processResmokeLine = (line: string) => {
+  // First check if it is a resmoke line at all
+  const resmokeFunction = getResmokeFunction(line);
+  if (!resmokeFunction) {
+    return line;
+  }
+  // Try to get the JSON string in the resmoke line
+  // If there is no JSON string, return the line as is
+  const json = getJSONString(line);
+  const port = getPort(line);
+  if (!json) {
+    return line;
+  }
+
+  const timeStamp = getTimeStamp(json);
+  const shellPrefix = getShellPrefix(json)?.padEnd(2, " ");
+  const configSrv = getConfigServer(json)?.padEnd(8, " ");
+  const id = getId(json)?.padEnd(7, " ");
+  const ctx = getContext(json);
+  const msg = getMessage(json);
+  const attr = getAttributes(json);
+
+  const output = `${resmokeFunction} ${port ?? ""}| ${timeStamp ?? ""} ${
+    shellPrefix ?? ""
+  } ${configSrv ?? ""} ${id ?? ""} [${ctx ?? ""}] "${msg ?? ""}"${
+    attr ? `,${attr}` : ""
+  }`;
+  return output;
+};
+
```

Comment:
Do we need to extract whatever these `truncated` and `size` values are, or do they seem like deprecated stuff?

https://github.com/evergreen-ci/lobster/blob/9039b4a104689afc5c86d46fd869397f8deea239/src/selectors/lines/transforms.js#L26-L29

Reply:
I couldn't actually find any logs with these fields so i'm assuming its possibly something legacy? I think it might be ok to ignore them for now and if we realize we need them it should be easy to add them back. It's difficult to craft a regex for them if i don't have a reference.